### PR TITLE
help: improve Symbol docs

### PR DESCRIPTION
**Pull request by snappizz**
clarify the differences between symbol and sring

### CODE REVIEW

File: HelpSource/Classes/Symbol.schelp [hunk 1]
```diff
@@ -1,10 +1,16 @@
 class::Symbol
-summary::unique name
+summary::a name for something
 categories::Core
 
 description::
-A Symbol is a name that is guaranteed to be unique. They can be used to represent
-symbolic constant values, link::Classes/Dictionary:: keys, etc.
+A symbol, like a link::Classes/String::, is a sequence of characters. Unlike strings, two symbols
+with exactly the same characters will be the exact same object. Symbols are optimized for recreating
```
Comment (telephon) [inline HelpSource/Classes/Symbol.schelp]: there seem to be unnecessary newlines

File: HelpSource/Classes/Symbol.schelp [hunk 2]
```diff
@@ -1,10 +1,16 @@
 class::Symbol
-summary::unique name
+summary::a name for something
```
Comment (brianlheim) [inline HelpSource/Classes/Symbol.schelp]: "unique" is a very important characteristic here, please don't remove that

File: HelpSource/Classes/Symbol.schelp (hunk 1 above)
Comment (brianlheim) [inline HelpSource/Classes/Symbol.schelp]: ?? we've agreed that 80-100-col style is ok for text-ish docs

File: HelpSource/Classes/Symbol.schelp (hunk 2 above)
Comment (snappizz) [inline HelpSource/Classes/Symbol.schelp]: i don't think it adds much, but if you insist

**Pull request merged**